sys/fs/tmpfs/tmpfs_subr.c

```diff
@@ -341,7 +341,8 @@ tmpfs_alloc_node(struct mount *mp, struct tmpfs_mount *tmp, enum vtype type,
 			NULL /* XXXKIB - tmpfs needs swap reservation */);
 		VM_OBJECT_WLOCK(obj);
 		/* OBJ_TMPFS is set together with the setting of vp->v_object */
-		vm_object_set_flag(obj, OBJ_TMPFS_NODE);
+		vm_object_set_flag(obj, OBJ_TMPFS_NODE | OBJ_NOLOADTAGS |
+		    OBJ_NOSTORETAGS);
```

Comment:
Did you actually test with a tmpfs? Like vm_object_clear_flag as mentioned on Slack, vm_object_set_flag also currently takes a u_short, so I would expect these flags to be lost. Please update the type in both of them.

Reply:
Ah Jenkins even turns this up: `error: implicit conversion from 'int' to 'u_short' (aka 'unsigned short') changes value from 197120 to 512 [-Werror,-Wconstant-conversion]`. Presumably you're doing a NO_MODULES build, and for a config that doesn't compile TMPFS into the kernel (ie not one of our std.MFS_ROOT ones).

Reply:
No, I haven't yet run this was going to let CI run it first overnight before coming back to it today.  I will address this today.